draft-ietf-diem-requirements.md

```diff
@@ -234,14 +234,27 @@ Similar use case as OPCW.
 ## International Humanitarian Law
 
 The Geneva Conventions and their Additional Protocols constitute the core of International Humanitarian Law (IHL).
-Some assets enjoy certain specific protections under IHL, including that they must not be attacked, and IHL codifies four types of protective emblems for armed conflict, which inform other parties that marked assets benefit from one or several of these specific protections:
+Under IHL, some assets enjoy certain specific protections under IHL, including that they must not be attacked, and IHL codifies four types of protective emblems for armed conflict, which inform other parties that marked assets benefit from one or several of these specific protections:
```

Comment:
this now says "under IHL" in two places. if the goal is to move it forward, perhaps we remove the second copy?

Reply:
Thanks!